pyproject.toml

```diff
@@ -24,6 +24,8 @@ python = ">=3.6.1,<4.0"
 luddite = "^1.0.1"
 packaging = "^20.8"
 pretty-errors = "^1.2.19"
+psycopg2-binary = ">=2.8.6"
```

Comment:
I also tested in a new venv with just the following libraries installed:
```
pip install sqlalchemy psycopg2 snowflake-sqlalchemy
```

And it worked. So because we already have a requirement for `psycopg2` and `psycopg2-binary` is not recommended for production:
> The binary package is a practical choice for development and testing but in production it is advised to use the package built from sources.
(as mentioned here: https://pypi.org/project/psycopg2-binary/)

I would advise to do `poetry remove psycopg2-binary`.

I suspect maybe you had tested without having made the venv properly with poetry maybe? Because for me both work totally fine.